Fix change_stored_password to support angular link

mfalesni opened the pull request:
Fix change_stored_password to support angular link
Also:
* Tweak the host analysis test code
* Fixup the blocker
* Handle double alerts

{{pytest: cfme/tests/infrastructure/test_host_analysis.py -v --long-running }}

dajoRH commented:
Lint report for commit 4aacd20f841dd82ad210e7e739604a62ade9a2f8:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

dajoRH commented:
Lint report for commit f30d4a0f66bff484e118524a136ebc8a535e1c23:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

dajoRH commented:
Lint report for commit 65b06eb950d03f833b1706e7882808721e008588:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

dajoRH commented:
Lint report for commit 31ac0347aeabeb1278e2eadb3dbbeadd8f4b40d7:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

mfalesni commented:
@psav Requesting review. I got good results last time, basically just one timeout in the tests, rest appeared to be working. So I just squashed it. I also did couple of changes to alert handling, it seems there were some additional alerts after the task deletion confirmation alert, so I added a functionality to just blow them away.

dajoRH commented:
Lint report for commit 19e0ae245f385d4ecdf8fde1827c8d4eceaf2476:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

dajoRH commented:
Lint report for commit 782eaefd204829d11f08a6cd5d3a576a3afbb38a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

mfalesni commented on cfme/fixtures/pytest_selenium.py:
```diff
@@ -484,6 +510,8 @@ def handle_alert(cancel=False, wait=30.0, squash=False, prompt=None):
             popup.send_keys(prompt)
         popup.dismiss() if cancel else popup.accept()
+        # Should any problematic "double" alerts appear here, we don't care, just blow'em away.
+        dismiss_any_alerts()
```
@psav I'd like to ask you what do you think of this? This might be causing some of the `h is null` errors like what I have fixed in this PR. I added a couple of new functions to support this functionality.
